docs/docs-beta/docs/guides/build/project-structure/components.md

```diff
@@ -8,27 +8,37 @@ Welcome to Dagster Components.
 
 Dagster Components is a new way to structure your Dagster projects. It aims to provide:
 
-- An opinionated project layout that supports ongoing scaffolding from "Hello world" to the most advanced projects
-- A class-based interface for dynamically constructing definitions
-- A toolkit to build YAML DSL frontends for components so that components can be constructed in a low-code fashion.
-- A format for components to provide their own scaffolding, in order to organize and reference integration-specific artifacts files.
+- An opinionated project layout that supports ongoing scaffolding from "Hello world" to the most advanced projects.
+- A class-based interface (`Component`) for dynamically constructing Dagster definitions.
+- A toolkit for buildng YAML DSLs for `Components`, allowing instances of components to be defined with little to no Python code.
```

Comment:
Do we want to mention the built-in set of Components for common use-cases/integrations,  as a goal?